src/table-view/table-view.jsx

```diff
@@ -0,0 +1,92 @@
+import React, { Fragment } from 'react'
+import createReactClass from 'create-react-class'
+import { withStyles, Button } from '@material-ui/core'
+import _ from 'lodash'
+import Select from '../select'
+import Table from '../table'
+
+const TableView = createReactClass({
+  displayName: 'TableView',
+  propTypes: {},
```

Comment:
Why is this empty?

Reply:
I did that because we were not getting anything from HOC. But now, we are getting the `selectTypes` from props. So, now we'll have `selectTypes` in the props.